scripts/login-handler/docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: '2'
+
+services:
+  login-handler:
+    container_name: tobira-login-handler-dummy
```

Comment:
```suggestion
```

How married are you to this line? In my experience, `container_name` makes using multiple (different versions) of this service harder. That includes scenarios like switching branches. It's nice for production environments, but I would leave it out for dev scripts like this.